jsonschema-validation.xml

```diff
@@ -575,21 +559,23 @@
                     validates against the provided schema.
                     Note the property name that the schema is testing will always be a string.
                 </t>
+                <t>
+                    A missing keyword has the same behavior as an empty schema.
```

Comment:
"empty schema" is language that was used before, can we work on this in a new PR and call this one good?